tables/topic_combinations.md

```diff
@@ -1 +1,56 @@
-topic_combinations.md
+# tables/topic_combinations
+
+
+## What is topic_combinations?
+The `topic_combinations` table lists out the available `topic` combinations for each possible set of `geography_combinations`.
+
+Once a user has selected what locations to filter by, they will then be prompted to select topics alongside corresponding filters. However the data has been setup so that only certain combinations of `variables` can be selected depending on what `geography_combinations` have been chosen. This is to stop users manipulating data to identify individuals using the data.
+
+## Example use
+Let's say that you have a topic within the `topics` table and you want to check what topic combinations that topic fits into
+
+```sql
+select ID, COMBINATION
+from topic_combinations
+where ARRAY['AGE'] <@ ARRAY[combination]
+```
+
+### Results
```

Comment:
Probably don't need the results header here (unless we change all the others)

Reply:
removed